packages/create-sitecore-jss/src/templates/nextjs/src/lib/page-props.ts

```diff
@@ -17,4 +17,5 @@ export type SitecorePageProps = {
   notFound: boolean;
   layoutData: LayoutServiceData;
   redirect?: Redirect;
+  headLinks: string[];
```

Comment:
See previous comment, this should be an array of a new type (with "rel" and "href" props).